releases/v19.1.6.md

```diff
@@ -0,0 +1,101 @@
+---
+title: What&#39;s New in v19.1.6
+toc: true
+summary: Additions and changes in CockroachDB version v19.1.6 since version v19.1.5
+---
+
+## December 16, 2019
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.1.6.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.1.6.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.1.6.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v19.1.6
+~~~
+
+
+### SQL language changes
+
+- [`EXPLAIN (OPT,ENV)`](../v19.1/explain.html) now returns a URL with the data encoded in the fragment portion. Opening the URL shows a page with the decoded data. Note that the data is processed in the local browser session and is never sent out. [#41092][#41092]
+- `EXPLAIN ANALYSE` can now be used as an alias to [`EXPLAIN ANALYZE`](../v19.1/explain-analyze.html). [#41093][#41093]
+- Mutations under `UNION` or `UNION ALL` are now disallowed. This restriction is temporary and will be lifted in a future release. [#41496][#41496]
+
+### Bug fixes
+
+- Fixed a crash that occurs when a suboperator with a `LIKE` comparison has a _NULL_ left-hand side. [#41073][#41073]
+- Reduced write-amp by avoiding forcing files through compactions unnecessarily. [#41301][#41301]
```

Comment:
~Suggest changing to "write amplification" so people can find it via search.  Most search is terribad at finding things that are not exact-match whole words.